server/lib/models/channel.go

```diff
@@ -1,7 +1,13 @@
 package models
 
 import (
-	proto "../../proto"
+	"log"
+	"bytes"
+	"fmt"
+	"math/rand"
+	"strings"
+
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)